integration/tests/main_test.go

```diff
@@ -0,0 +1,41 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+)
+
+var assetInventoryAPI *openapi.APIClient //this contains pre-configured API client used by all fixtures
+
+func TestMain(m *testing.M) {
+	config := openapi.NewConfiguration()
+	appURL := os.Getenv("AIA_APP_URL")
+	config.BasePath = appURL
+	_, config.Debug = os.LookupEnv("AIA_INTEGRATION_DEBUG")
+	assetInventoryAPI = openapi.NewAPIClient(config)
+	ctx := context.Background()
+	schemaVersion = getSchemaVersion(ctx, assetInventoryAPI.DefaultApi)
+	res := 0
+
+	//run all know tests with all supported schema versions
+	for v := int32(12); v <= maxSchema; v++ {
```

Comment:
Question: Does the order of integration test execution matter? I think we may face unexpected failure if we are running this test in conjunction with other tests.

Reply:
Schema changes definitely will result in failure if used with same database concurrently. So order of fixtures (i.e. schema generations) matter, or rather - only one can be used with same db. Order of tests inside fixture should not (all tests are run for every fixture). This way we can run multiple db instances concurrently with different fixtures (version of schema), but not with different schemas on same db concurrently.